src/main/assisted-installer-controller/assisted_installer_main.go

```diff
@@ -18,6 +22,11 @@ var Options struct {
 	ControllerConfig assistedinstallercontroller.ControllerConfig
 }
 
+var (
+	envProxyOnce          sync.Once
```

Comment:
why we need this const?

Reply:
in order to evaluate the proxy configuration only once after they were set. the same approach is used by `http.ProxyFromEnvironment`